workersController.js

```diff
@@ -19,94 +20,96 @@ var config = require('./config.json');
  * Function is invoked by SocketCluster
  * @param {Worker} worker
  */
-module.exports.run = function (worker) {
+class Worker extends SCWorker {
```

Comment:
The whole codebase is currently in ES5 code standards, so a user can still run Lisk from source on any Node.js version. Is it possible to set up `SCWorker` in a different way than extending a class?

Reply:
@MaciejBaj Yes it's possible, I'll change this.